### PR TITLE
dockercfg Generator

### PR DESCRIPTION
**Pull request by mlocher**
Add a image to generate credentials for the Heroku Container Registry, see https://devcenter.heroku.com/articles/container-registry-and-runtime for details.

### CODE REVIEW

File: dockercfg-generator/Dockerfile.test [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.4
+MAINTAINER maintainers@codeship.com
+
+RUN \
+  echo "Hello ECR at $(date)" > hello.txt && \
```
Comment (rheinwein) [inline dockercfg-generator/Dockerfile.test]: Maybe we could change this `hello` message to be less specific (or hello heroku, instead)?

Comment (mlocher) [reply dockercfg-generator/Dockerfile.test]: Absolutely, I missed changing this :(

**Comment by rheinwein** [comment]
Nice 👍 

Should we rename this repo in the same way we're renaming the AWS repo?

**Comment by mlocher** [comment]
Yeah, I'd rename the repository to `heroku-utilities` and also prepare a PR to move the deployment container source files into a subfolder.

**Pull request merged**